components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt

```diff
@@ -331,7 +336,12 @@ class AddonsManagerAdapter(
         val addonSummaryTextColor: Int? = null,
         val sectionsTypeFace: Typeface? = null,
         @DrawableRes
-        val addonAllowPrivateBrowsingLabelDrawableRes: Int? = null
+        val addonAllowPrivateBrowsingLabelDrawableRes: Int? = null,
+        val visibleDividers: Boolean = true,
+        @ColorRes
+        val dividerColor: Int? = null,
+        @DimenRes
+        val dividerHeight: Int? = null
```

Comment:
Maybe we could create a `Style.CategoryDivider` data class that it will contain these values, and future once. This way we could avoid continuing growing the `Style` parameters and avoid ambiguity between passing each to of the values separated.

For example: Now it's possible to have:
`visibleDividers = true`, `dividerColor = null` and `dividerHeight: Int? = null`

Which colour and height the divider should have?

Another example:
 `visibleDividers = true`, `dividerColor = null` and `dividerHeight: Int? R.dimen.mozac_browser_menu_item_divider_height`

Which colour the divider should have?

If we have a `CategoryDivider` optional(?) reference, on Style, we can have all these values on it and if it set to null we can assume that the consumer doesn't want dividers in the categories.

What do you think? :)

Reply:
This was my main concern for this: if it is better to use a style for dividers or separate attributes.
On the one hand, we have fewer parameters in style, on the other hand, using a style we make the passing of a single change (maybe we want to change just the height) more complicated.
I am not sure which one is better, I am ok, either way.

Reply:
> For example: Now it's possible to have:
> `visibleDividers = true`, `dividerColor = null` and `dividerHeight: Int? = null`
> 
> Which colour and height the divider should have?

It would have the default colour and height.
> 
> Another example:
> `visibleDividers = true`, `dividerColor = null` and `dividerHeight: Int? R.dimen.mozac_browser_menu_item_divider_height`
> 
This would make possible changing only the height, without having to pass any other attributes.
> 
> If we have a `CategoryDivider` optional(?) reference, on Style, we can have all these values on it and if it set to null we can assume that the consumer doesn't want dividers in the categories.
> 
> What do you think? :)

 Setting the values to null (or just not passing values and defaulting to null values) means we do not want to change default style.

Reply:
We can keep it as it's 👍🏽